index.html

```diff
@@ -89,109 +85,25 @@ <h3><em>Convert a Scratch project to HTML</em></h3>
-      <h3>2020-06-01</h3>
-      <ul>
-        <li>Option to remove clone/list length limits</li>
-        <li>Option to hide the cursor</li>
-        <li>Ability to distinguish between cloud variables for localStorage and the server (and run JavaScript using cloud variables)</li>
-        <li>Option for rudimentary pointer lock</li>
-        <li>Fixed mouse position and unresized lists not showing</li>
-        <li>Default project changed to one of <a href="https://scratch.mit.edu/users/ScratchCat/">ScratchCat</a>'s</li>
-        <li>Unfortunately, as part of these changes, I removed the option for a progress bar. If there's demand, I'll try to add it back.</li>
-      </ul>
-      <h3>2020-05-01</h3>
-      <ul>
-        <li>Support for custom extensions from a URL</li>
-        <li>CSS by <a href="https://scratch.mit.edu/users/mrcringekidyt/">Mr. Cringe Kid</a></li>
-      </ul>
-      <h3>2020-03-29</h3>
-      <ul>
-        <li>Fixed custom stage sizes</li>
-      </ul>
-      <h3>2020-03-27</h3>
-      <ul>
-        <li>Show an image such as a gif while loading</li>
-        <li>Fixed dragging sprites simulating another green flag click</li>
-      </ul>
-      <h3>2020-01-06</h3>
-      <ul>
-        <li>An offline version of the HTMLifier</li>
-      </ul>
-      <h3>2019-12-25</h3>
-      <ul>
-        <li>An option to use a custom cloud server for cloud variables instead of saving to localStorage</li>
-      </ul>
-      <h3>2019-11-23</h3>
-      <ul>
-        <li>A fullscreen button</li>
-        <li>An option to change monitor colours</li>
-      </ul>
-      <h3>2019-10-05</h3>
-      <ul>
-        <li>New ask box</li>
-      </ul>
-      <h3>2019-09-28</h3>
-      <ul>
-        <li>A status text that shows the assets loaded</li>
-        <li>Support the video extension</li>
-        <li>Support draggable sprites</li>
-      </ul>
-      <h3>2019-08-08</h3>
-      <ul>
-        <li>Removed "Scratch" from the name of this utility just in case</li>
-      </ul>
-      <h3>2019-07-27</h3>
-      <ul>
-        <li>Support 16:9 projects</li>
-        <li>Simplify the number of modes for HTMLification</li>
-      </ul>
-      <h3>2019-07-13</h3>
-      <ul>
-        <li>Cloud variables store in localStorage</li>
-      </ul>
-      <h3>2019-06-29</h3>
-      <ul>
-        <li>Upload project file instead of using project ID</li>
-        <li>Toggle compatibility and turbo mode</li>
-      </ul>
-      <h3>2019-03-31</h3>
-      <ul>
-        <li>Variable and list watchers</li>
-      </ul>
-      <h3>2019-02-09</h3>
-      <ul>
-        <li>Project start</li>
-      </ul>
+      
       <p>Made by <a href="https://scratch.mit.edu/users/Sheep_maker/">Sheep_maker</a>, who used <a href="https://github.com/LLK/scratch-vm/">scratch-vm</a>, <a href="http://danml.com/download.html">download.js</a>, and their dependencies for this project.</p>
-      <p>CSS by <a href="https://scratch.mit.edu/users/mrcringekidyt/">Mr. Cringe Kid</a>.</p>
```

Comment:
It seems some portion of the CSS is still theirs; I wouldn't remove their credit

Reply:
True.